routers/api/v1/api.go

```diff
@@ -1172,6 +1176,10 @@ func Routes(ctx gocontext.Context) *web.Route {
 				m.Get("/issue_config", context.ReferencesGitRepo(), repo.GetIssueConfig)
 				m.Get("/issue_config/validate", context.ReferencesGitRepo(), repo.ValidateIssueConfig)
 				m.Get("/languages", reqRepoReader(unit.TypeCode), repo.GetLanguages)
+
+				m.Group("/activities", func() {
+					m.Get("/feeds", repo.ListRepoActivityFeeds)
+				})
```

Comment:
```suggestion
				m.Group("/activities/feeds", repo.ListRepoActivityFeeds)
```

Here and below, like this?